pkg/api/config.go

```diff
@@ -19,27 +19,7 @@ type Config struct {
 	ImageResourceName  string `json:"imageResourceName,omitempty"`
 
 	Certificates CertificateConfig `json:"certificates,omitempty"`
-
-	// container images for pods
-	MasterEtcdImage             string `json:"masterEtcdImage,omitempty"`
-	ControlPlaneImage           string `json:"controlPlaneImage,omitempty"`
-	NodeImage                   string `json:"nodeImage,omitempty"`
-	ServiceCatalogImage         string `json:"serviceCatalogImage,omitempty"`
-	SyncImage                   string `json:"syncImage,omitempty"`
-	TemplateServiceBrokerImage  string `json:"templateServiceBrokerImage,omitempty"`
-	PrometheusNodeExporterImage string `json:"prometheusNodeExporterImage,omitempty"`
-	RegistryImage               string `json:"registryImage,omitempty"`
-	RouterImage                 string `json:"routerImage,omitempty"`
-	AzureCLIImage               string `json:"azureCliImage,omitempty"`
-	RegistryConsoleImage        string `json:"registryConsoleImage,omitempty"`
-	AnsibleServiceBrokerImage   string `json:"ansibleServiceBrokerImage,omitempty"`
-	WebConsoleImage             string `json:"webConsoleImage,omitempty"`
-	OAuthProxyImage             string `json:"oAuthProxyImage,omitempty"`
-	PrometheusImage             string `json:"prometheusImage,omitempty"`
-	PrometheusAlertBufferImage  string `json:"prometheusAlertBufferImage,omitempty"`
-	PrometheusAlertManagerImage string `json:"prometheusAlertManagerImage,omitempty"`
-	LogBridgeImage              string `json:"logBridgeImage,omitempty"`
-	EtcdOperatorImage           string `json:"etcdOperatorImage,omitempty"`
+	Images       ImageConfig       `json:"images,omitempty"`
```

Comment:
property name suggests this should be a list, shouldn't this be ``[]ImageConfig`` instead?